sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/tests/Listeners/ServiceBusScaleMonitorTests.cs

```diff
@@ -122,31 +123,31 @@ public void GetMetrics_ReturnsExpectedResult()
             var message = ServiceBusModelFactory.ServiceBusReceivedMessage(enqueuedTime: utcNow);
 
             // Test base case
-            var metrics = ServiceBusScaleMonitor.CreateTriggerMetrics(null, 0, 0, 0, false);
+            var metrics = ServiceBusMetricsReceiver.CreateTriggerMetrics(null, 0, 0, 0, false);
```

Comment:
After our refactoring these methods have moved to a new type. We should move the corresponding tests into a new ServiceBusMetricsProvider test suite